lib/tasks/grade.rake

```diff
@@ -92,6 +97,42 @@ namespace :grade do
 
 end
 
+def sync_specs_with_source
+  reponame = Dir.pwd.split("/").last
```

Comment:
I think this is fine for now. But in the long-run, it seems a bit brittle; what if they named their repo something else, or `git cloned` to a differently named folder?

Perhaps a combo of Octokit + `git remote -v` could be used to detect the upstream repo more reliably. But not a big deal for now

Reply:
> what if they named their repo something else, or git cloned to a differently named folder?

I assumed this would be fine for the foreseeable future of students using Gitpod, or at least Grades- which would name the directory appropriately. But you're right, a folder rename would break this.

Reply:
Also, if they aren't using Gitpod then wouldn't we be unable to determine the upstream branch regardless? I think if you just clone a forked repo, you don't automatically get the upstream branch.